app/models/expense.rb

```diff
@@ -7,4 +7,22 @@ class Expense < ActiveRecord::Base
   validates :description, :user, :amount, :paid_on, :category, presence: true
   validates :paid_on, date: { before_or_equal_to: proc { Date.today } }
   validates_associated :receipt
+
+  def format_amount
+    '€' + amount.to_s
+  end
+
+  def format_vat
+    vat.to_s + '%' unless vat.nil?
+  end
+
+  def format_amount_w_vat
+    '€' + amount_w_vat.to_s unless vat.nil?
+  end
```

Comment:
I'm not sure about these three methods. I don't think they should be here because they are view-related, but were should I place them?